### PR TITLE
Feature/python3

### PR DESCRIPTION
**Pull request by Lujeni**

### CODE REVIEW

**Comment by Lujeni** [comment]
@ramnes @ultrabug thanks

**Comment by ramnes** [comment]
LGTM

File: mattermost_giphy/app.py [hunk 1]
```diff
@@ -1,19 +1,25 @@
 # -*- coding: utf-8 -*-
-import logging
-import os
-import sys
+
```
Comment (ramnes) [inline mattermost_giphy/app.py]: nop

File: mattermost_giphy/app.py [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 import requests
 from flask import Flask
-from flask import request
 from flask import Response
+from flask import request
 
-from mattermost_giphy.settings import *
+from mattermost_giphy.settings import (
+        USERNAME, ICON_URL, RATING, SCHEME, GIPHY_API_KEY,
+        MATTERMOST_GIPHY_TOKEN)
```
Comment (ramnes) [inline mattermost_giphy/app.py]: it would look better to put the closing parenthesis on a new line

Comment (ultrabug) [reply mattermost_giphy/app.py]: if we want to address coding style, let's do it once and for all with CI, but that's not the purpose of this PR I guess

**Comment by ultrabug** [comment]
LGTM, thanks for this!

File: mattermost_giphy/app.py (hunk 2 above)
Comment (ramnes) [reply mattermost_giphy/app.py]: hence me saying LGTM before, as well as putting a single comment and not a review :wink:

Comment (ultrabug) [reply mattermost_giphy/app.py]: <3

**Pull request merged**